diagnostic.md

```diff
@@ -69,13 +66,13 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 ```
 
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
-
- <!-- Answer Starts Here -->
+relative
+ <!-- Answer Starrelativets Here -->
 
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
-
+relative
```

Comment:
Correct, but it asks you to explain your answer: the image file lives inside our current working directory, project; as a result, an absolute path is not needed to access it.